.github/workflows/build-and-test.yml

```diff
@@ -19,7 +19,7 @@ jobs:
   post-pull-request-checks-automation:
     needs: [call-build-and-test-workflow]
```

Comment:
Please put this step after "Spelling", and add `spelling` to `needs`, just as here: https://github.com/Lombiq/Open-Source-Orchard-Core-Extensions/blob/dev/.github/workflows/build-and-test.yml#L41-L51